<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_interface</class>
 <widget class="QWidget" name="GUI_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI_interface</string>
  </property>
  <widget class="QLabel" name="labelX">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelY">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelZ">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-axis</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="ScrollBarX">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="ScrollBarY">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="ScrollBarZ">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonUpdateValeus">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Values</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxX">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLights">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Lights On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLand">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Land</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxY">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxZ">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>200</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSetNewValues">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>37</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set New Values</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSTOP">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Emergency Stop</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCurrentPosition">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="PositionX">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="positionY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="positionZ">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>490</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="ConfidenceX">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="ConfidenceY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="ConfidenceZ">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentX">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentY">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentZ">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentPositionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfidenceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confidence</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="XY_diagram">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
